scripts/serve-local.sh

```diff
@@ -1,14 +1,24 @@
 #!/usr/bin/env bash
+echo "starting..."
 
 set -o errexit
 
-TAG='janis:local'
+echo "fetching node packages on host..."
+mkdir -p .yarn-cache
+yarn install --cache-folder .yarn-cache
```

Comment:
We run the risk of building something for the host machine that isn't compatible with the docker OS. I'm starting to think the benefits of the cache on the host isn't that beneficial compared to the work involved to make it happen.